src/frontend/screens/MapScreen/CurrentTrack/CurrrentTrackMapLayer.tsx

```diff
@@ -6,7 +6,7 @@ import {LineString} from 'geojson';
 import {useLocation} from '../../../hooks/useLocation';
 import {LocationHistoryPoint} from '../../../sharedTypes/location';
 
-export const TrackPathLayer = () => {
+export const CurrentTrackMapLayer = () => {
```

Comment:
I renamed this so there was a distinction between the track being made and previous tracks shown on the map